pkg/scheduler/plugins/rescheduling/node_utilization_util.go

```diff
@@ -207,9 +208,9 @@ func convertPercentToQuan(rName v1.ResourceName, percent float64, nodeCapacity v
 func convertQuanToPercent(rName v1.ResourceName, amount *resource.Quantity, nodeCapacity v1.ResourceList) float64 {
 	var percent float64
 	if rName == v1.ResourceCPU {
-		percent = amount.AsApproximateFloat64() / nodeCapacity.Cpu().AsApproximateFloat64()
+		percent = amount.AsApproximateFloat64() * 100 / nodeCapacity.Cpu().AsApproximateFloat64()
```

Comment:
Thanks for the fix. Can you provide more details about the datas `amount` and `percent` you tested locally? Perhaps I miss something and it seems correct datas when I tested locally.